scripts/cal_acc.py

```diff
@@ -13,27 +13,32 @@
 #  limitations under the License.
 import sys
 import argparse
-from evaluator import evaluate
+from ns_evaluator import LMEvalParser
+from accuracy import cli_evaluate
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(description="Evaluate accuracy for a model")
     parser.add_argument('--model_name', type=str, default="~/Llama-2-7b-chat-hf")
     parser.add_argument('--tasks', type=str, default="lambada_openai")
-    parser.add_argument('--model_format', type=str, default="runtime")
+    parser.add_argument('--model_format', type=str, default="neural_speed")
     parser.add_argument("--use_gptq", action="store_true")
     parser.add_argument("--use_awq", action="store_true")
     parser.add_argument("--use_autoround", action="store_true")
+    parser.add_argument('--batch_size', type=int, default=1)
     args = parser.parse_args()
 
-    model_name = args.model_name
-    model_format = args.model_format
-    tasks = args.tasks
-    results = evaluate(
-        model="hf-causal",
-        model_args=f'pretrained="{model_name}",use_gptq={args.use_gptq},use_awq={args.use_awq},use_autoround={args.use_autoround}',
-        tasks=[f"{tasks}"],
-        # limit=5,
-        model_format=f"{model_format}"
-    )
+    model_args=f'pretrained={args.model_name},dtype=float32,trust_remote_code=True'
```

Comment:
I suggest adding more quant args (like `weight_dtype`, `group_size`, etc.) here so that we can evaluate any kind of quantized ns model. cc @a32543254